root/static/scripts/account/components/PreferencesForm.js

```diff
@@ -231,6 +232,15 @@ class PreferencesForm extends React.Component<Props, State> {
             )}
             uncontrolled
           />
+          <FormRowCheckbox
+            field={field.skip_email_on_abstain}
+            label={l(
+              `When I vote Abstain on an edit,
+               do not mail me all future notes for that edit, even if
+               I usually get mailed on votes.`,
+            )}
+            uncontrolled
+          />
```

Comment:
I considered this, but then I remembered Approve is also a vote and it started to feel silly :thinking:

Reply:
A little, but I guess we could have a separate string with "Approve" listed only for auto-editors. For normal users it might be easier to understand than the current negated option. :) Though I am fine with either if you prefer to keep it as-is.

Reply:
Hmm, I think I prefer it split out. It's quite a confusing concept, and bitmap's suggestion definitely helped me grok it faster. I don't think it's that bad if option A includes 'Approve'.

I looked at some other subscription thingies online and this seems to be pretty standard, wording wise:

Email me a notification when:
[ ] There is activity on an edit I have voted on
[ ] There is activity on an edit I have abstained on

Reply:
I did basically as aerozol suggested, including shortening the lines in a separate commit:

![Screenshot from 2023-12-08 12-31-40](https://github.com/metabrainz/musicbrainz-server/assets/1069224/da006d58-378a-4655-990e-0fc061d76caa)

I also made it default to `email_on_vote` - I hope correctly, tested a bit and it did seem right at least.